<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search</class>
 <widget class="QWidget" name="search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>根据不同种类进行查找</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="idInput"/>
            </item>
            <item>
             <widget class="QLineEdit" name="genusInput"/>
            </item>
            <item>
             <widget class="QLineEdit" name="ageInput"/>
            </item>
            <item>
             <widget class="QLineEdit" name="sexInput"/>
            </item>
            <item>
             <widget class="QLineEdit" name="statusInput"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="searchByid">
              <property name="text">
               <string>id</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="searchBygenus">
              <property name="text">
               <string>种类</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="searchByage">
              <property name="text">
               <string>年龄</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="searchBysex">
              <property name="text">
               <string>性别</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="searchBystatus">
              <property name="text">
               <string>状态</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="ShowAllButton">
           <property name="text">
            <string>显示全部</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ResetButton">
           <property name="text">
            <string>resetID</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
